recipe/meta.yaml

```diff
@@ -12,13 +12,15 @@ source:
         - make_check.patch  # [osx]
 
 build:
-    number: 1
+    number: 2
     features:
-     - vc9  # [win and py27]
-     - vc10  # [win and py34]
-     - vc14  # [win and py35]
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
 
 requirements:
+    build:
+        - python  # [win]
```

Comment:
No, it's different from putting Python in the runtime requirements (which is when that gets pinned.)

This is telling it to pay attention to which Python is being used to build.  Without it, I don't think the selectors are activated.  I might be wrong, though.